src/main/java/com/courtcanva/ccfranchise/exceptions/NoSuchElementException.java

```diff
@@ -0,0 +1,8 @@
+package com.courtcanva.ccfranchise.exceptions;
+
+public class NoSuchElementException extends RuntimeException {
+
+    public NoSuchElementException(String msg) {
+        super(msg);
```

Comment:
Maybe you can use the existing exception(ResourceNotFoundException).

Reply:
Done, this is updated.